client/src/app/components/Sidebar/index.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export default () => {
+  return (
+    <div className='flex flex-col py-4 items-center'>
```

Comment:
🔮  Can we make the sidebar either obviously the same color or obviously a different color (like a light gray?) than the Navbar?

https://github.com/joshprzybyszewski/cribbage/blob/892fbf53fcbdcd3d5adb6cc8c7920b5cc477faba/client/src/app/containers/Navbar/index.jsx#L20

Reply:
Yap, I can play around with colors later today.